docs/ansible/ansible.md

```diff
@@ -14,6 +14,7 @@ python3 -m venv $VENVDIR
 source $VENVDIR/bin/activate
 cd $KUBESPRAYDIR
 pip install -U -r requirements.txt
+pip install -U -r contrib/inventory_builder/requirements.txt
```

Comment:
You don't need this if you don't use the dynamic inventory, please make this clear

Reply:
I did what https://github.com/kubernetes-sigs/kubespray/blob/1798989f9918b23bcfca88c29e5af6875ed0502f/README.md?plain=1#L29-L33 describes. 
Executing `CONFIG_FILE=inventory/z2air/hosts.yaml python contrib/inventory_builder/inventory.py ${IPS[@]}` gives the error `Not a module named ruamel`. 

Looking through the issue, I found a solution that requires the `ruamel.yaml` module to be installed.

A number of people have had the same problem, and I think it's necessary to add the relevant description somewhere correct, but I'm not sure I'm adding it in the right place.

Reply:
Then this line should be in the README just before using the inventory.py script, since it's not for installing ansible, but just to run the script.